api/controllers/case.js

```diff
@@ -226,6 +238,7 @@ router.post("/new", function(req, res, next) {
     })
     .catch(function(error) {
       log.error("Exception in POST /case/new => %s", error);
+      console.error(error);
```

Comment:
Oh, one can be removed. I added the console error because I wasn't seeing the log when running tests that errored out, but I didn't see the console message either.